src/ert/shared/services/_base_service.py

```diff
@@ -312,12 +312,9 @@ def connect(
             timeout = 120
         t = -1
         while t < timeout:
-            search_path = Path(path)
-            while search_path != Path("/"):
-                if (search_path / name).exists():
-                    with (search_path / name).open() as f:
-                        return cls([], conn_info=json.load(f))
-                search_path = search_path.parent
+            if (path / name).exists():
+                with (path / name).open() as f:
+                    return cls([], conn_info=json.load(f), project=path)
 
```

Comment:
we don't need to search upwards anymore? is that because we have more confidence about where we should find the file? or was the search strictly speaking not really necessary before, either?

Reply:
I don't think it was really necessary before also very prone to issues.

Reply:
gotcha!